src/goto-cc/compile.cpp

```diff
@@ -391,8 +391,9 @@ bool compilet::compile()
     source_files.pop_front();
 
     // Visual Studio always prints the name of the file it's doing
+    // onto stdout. The name of the directory is stripped.
     if(echo_file_name)
-      status() << file_name << eom;
+      std::cout << get_base_name(file_name, false) << '\n' << std::flush;
```

Comment:
Guessing this is only done for want of a `flush()` method on `messaget`, as I guess our driver knows that the messaget needs to point at stdout?

Reply:
No, this is really meant to ensure that the message goes to stdout, irrespectively of level settings etc

Reply:
Excuse me for wading in here, but this essentially breaks `--json-ui` - don't know if that's a concern here but makes it harder to use this from a calling program. May at least want to warn if json ui is enabled on the message_handler?

Reply:
goto-cc does not have the json ui option.

Reply:
Ah ok, so maybe just add a highest prio level to `messaget`? Or an alias for `error()` if that can never be silenced even with `--verbosity 0`?

Reply:
No, won't work -- the spec says to output to stdout, and error() will output to stderr.

Reply:
How about using the `cout_message_handlert`?